src/T2DMap.cpp

```diff
@@ -2636,21 +2649,28 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
 
             TRoom* pR = mpMap->mpRoomDB->getRoom(mCustomLinesRoomFrom);
             if (pR) {
-                QAction* action = new QAction("undo", this);
-                action->setToolTip(tr("Undo last point"));
+                QAction* action = new QAction(tr("undo",
+                                                 "2D Mapper context menu (drawing custom exit line) item"), this);
+                action->setToolTip(tr("Undo last point",
+                                      "2D Mapper context menu (drawing custom exit line) item tool-tip"));
                 if (pR->customLines.value(mCustomLinesRoomExit).count() > 1) {
                     connect(action, SIGNAL(triggered()), this, SLOT(slot_undoCustomLineLastPoint()));
                 } else {
                     action->setEnabled(false);
                 }
 
-                QAction* action2 = new QAction("properties", this);
-                action2->setText("properties...");
-                action2->setToolTip(tr("Change the properties of this line"));
+                QAction* action2 = new QAction(tr("properties",
+                                                  "2D Mapper context menu (drawing custom exit line) item name (but not used as display text as that is set separately)"), this);
```

Comment:
TBH I'm not 100% sure, it might get converted into the *objectName* as well - the documentation says:
> `QAction::QAction(const QString &text, QObject *parent = nullptr)`
Constructs an action with some *text* and *parent*. If *parent* is an action group the action will be automatically inserted into the group.
The action uses a stripped version of text (e.g. "&Menu Option..." becomes "Menu Option") as descriptive text for tool buttons. You can override this by setting a specific description with `setText()`. The same text will be used for tooltips unless you specify a different text using `setToolTip()`.